.coafile

```diff
@@ -28,6 +28,11 @@ bears = PyUnusedCodeBear
 default_actions =
     PyUnusedCodeBear: ApplyPatchAction
 
+[commit]
+bears = GitCommitBear
+shortlog_trailing_period = false
+shortlog_regex = ([^:]*: [A-Z].*)
```

Comment:
coala and coala-bears use [this](https://github.com/coala/coala/blob/master/.coafile#L64). Any reason for a different regex?